src/Npgsql/Replication/TestDecoding/TestDecodingAsyncEnumerable.cs

```diff
@@ -35,51 +35,47 @@ sealed class TestDecodingAsyncEnumerable : IAsyncEnumerable<TestDecodingData>
 
     public IAsyncEnumerator<TestDecodingData> GetAsyncEnumerator(CancellationToken cancellationToken = default)
     {
-        using (NoSynchronizationContextScope.Enter())
-        {
-            return StartReplicationInternal(
-                CancellationTokenSource.CreateLinkedTokenSource(_baseCancellationToken, cancellationToken).Token);
-        }
-    }
+        return StartReplicationInternal(CancellationTokenSource.CreateLinkedTokenSource(_baseCancellationToken, cancellationToken).Token);
```

Comment:
Can be inlined?

Reply:
This one can be changed indeed, just a bit awkward with the linkedtoken call in there

Reply:
Done